integrationTests/vm/testInitializer.go

```diff
@@ -563,7 +564,7 @@ func CreateVMAndBlockchainHookAndDataPool(
 		AutomaticCrawlerAddress:   bytes.Repeat([]byte{1}, 32),
 		MaxNumNodesInTransferRole: 100,
 	}
-	builtInFuncs, nftStorageHandler, globalSettingsHandler, _ := builtInFunctions.CreateBuiltInFuncContainerAndNFTStorageHandler(argsBuiltIn)
+	builtInFuncFactory, _ := builtInFunctions.CreateBuiltInFunctionsFactory(argsBuiltIn)
```

Comment:
naming: this is rather a handler, not a factory :D

Reply:
FactoryHandler to be precise 📦

Reply:
no, handler, as it stores components. Factory just creates pointers but do not hold them